department-of-veterans-affairs/caseflow
# Div/appeals 56950

Resolves [Restricting Params in appeals_controller.rb Causes Issues with adding unidentified issues](https://jira.devops.va.gov/browse/APPEALS-56950)

# Description
Removed Restricting Params in appeals_controller.rb file

## Acceptance Criteria

## Testing Plan


1. Log in as CF_Q_283 in UAT
2. Navigate Intake
3. Select 10182 from the options and click continue
4. Enter a veteran file number and click search
5. Fill out the form and click submit
6. On the Add issue page, click add issue and enter an issue
7. Click Establish
8. Click the correct issues link
9. On the correct issues page, click Add Issue
10. Click "none of these match". If needed, click "none of these match" again to get to the Unidentified Issue modal.
11. Fill out the description of the issue and click add.
12. Click save. 
13. Notice issue should be added


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
